synfig-studio/plugins/view-unhide-all-layers/CMakeLists.txt

```diff
@@ -0,0 +1,35 @@
+set(INSTALL_DESTINATION "share/synfig/plugins/view-unhide-all-layers")
+set(PLUGIN_VIEW_UNHIDE_ALL_LAYERS_FILES
+    view-unhide-all-layers.py
+)
+
+file(COPY ${PLUGIN_VIEW_UNHIDE_ALL_LAYERS_FILES}
+    DESTINATION ${SYNFIG_BUILD_ROOT}/${INSTALL_DESTINATION}
+)
+
+install(FILES ${PLUGIN_LOTTIE_EXPORTER_SYNFIG_FILES}
+    DESTINATION ${INSTALL_DESTINATION}
+)
+
+find_program(INTLTOOL_MERGE intltool-merge)
```

Comment:
Better to make a function for intltool and use it everywhere. 
Also better to add a warning instead of error if intltool not found, so code can be compiled, just without translations.